android/assets/po/es.po

```diff
@@ -832,12 +838,15 @@ msgid "6th"
 msgstr "6.º"
 
 #: desktop/src/com/agateau/pixelwheels/desktop/DesktopLogExporter.java:44
-msgid "Need to file a bug report? Use this button to find the log file to attach."
-msgstr "¿Necesitas crear un informe de error? Utiliza este botón para encontrar el fichero de registro a adjuntar."
+msgid ""
+"Need to file a bug report? Use this button to find the log file to attach."
+msgstr ""
+"¿Necesitas crear un informe de error? Utiliza este botón para encontrar el"
+" fichero de registro a adjuntar."
 
 #: desktop/src/com/agateau/pixelwheels/desktop/DesktopLogExporter.java:49
 msgid "OPEN LOG FILE FOLDER"
-msgstr "FICHERO DE REGISTRO"
+msgstr "ABRIR CARPETA DE ARCHIVO DE REGISTRO"
```

Comment:
You can add a `\n` to ensure the text fit in the button, to avoid this:

![screenshot-2022-09-10-194239 712](https://user-images.githubusercontent.com/3575/189495414-d76768a5-24cb-44ed-bb44-859ea384ffa7.png)


```suggestion
msgstr "ABRIR CARPETA DE\nARCHIVO DE REGISTRO"
```